scripts/deploy.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+set -ev
+testchange=`git diff --exit-code -- docs > /dev/null`
+if [$testchange]; then
```

Comment:
Should the test be `if [ $? -ne 0 ]` ? `$testchange` should always be empty.

Reply:
okay, I changed it - but then we have to also remove the "set -ev" line else we will exit if the command returns a non-zero status code, and Travis is adamant that we not call exit.

Reply:
FWIW: my test showed that testchange was also either equal to 0 or 1, not empty